### ISSUE TITLE
Adding multiple items from AD in mappings to insert into fe_users not possible

### ISSUE THREAD

**Issue by angeloprevitali** [xperseguers/t3ext-ig_ldap_sso_auth#40]
Hi Xavier

we would like to insert into the Page TS of the fe_users the two lines inserted like that if the be_user adds a new page the information of the page gets autocompleted.

`TCAdefaults.pages.author = Nom du auteur
TCAdefaults.pages.author_email = redacteur@netconsult.ch`

![be_users-pagets](https://user-images.githubusercontent.com/17588253/46476282-08df6b80-c7e8-11e8-9c35-d8c1eae20b1a.png)

If we insert this in the configuration of the LDAP / SSO > BE USERS field "Mapping" the second line gets not inserted into the config of the Page TS of the fe_users.

`tstamp = {DATE}
usergroup = <memberOf>
TSconfig = TCAdefaults.pages.author = <name>
TSconfig = TCAdefaults.pages.author_email = <mail>`

![ldap-pagets-settings-1](https://user-images.githubusercontent.com/17588253/46476288-0e3cb600-c7e8-11e8-9d8c-ec12ebb7b06f.png)

Any idea how to setup this? Is it possible?

Thanks for help.

Best regards,
Angelo

**Comment by xperseguers** [comment]
Probably not the easiest way but I can think of either custom marker (with an extension of your own to post-process the mapping) or using a TS-based mapping. See the doc for examples.

I agree, simple multi-line mapping is not possible out of the box.

**Comment by angeloprevitali** [comment]
Thanks. This code gets inserted into the Mapping field configuration of the LDAP / SSO > BE USERS field "Mapping"? right?

**Comment by xperseguers** [comment]
Exactly, please see https://docs.typo3.org/typo3cms/extensions/ig_ldap_sso_auth/AdministratorManual/Users.html#mapping

you may mix the various types of mapping definitions.

**Comment by angeloprevitali** [comment]
Thanks Xavier! I will try this!
How much will cost me this information?
And by the way: Your Google Maps on your website isn't working anymore: See https://www.causal.ch/#contact
Best regards,
Angelo

**Comment by xperseguers** [comment]
oops, thx :)

**Comment by xperseguers** [comment]
(info free of use) :)

**Comment by angeloprevitali** [comment]
thx Xavier! Salutations à Fribourg!
